docs/releases/unreleased.md

```diff
@@ -9,6 +9,10 @@
 
 - The `predict_many` method scikit-learn models wrapped with `compat.convert_sklearn_to_river` raised an exception if the model had been fitted on any data yet. Instead, default predictions will be produced, which is consistent with the rest of River.
 
+## neighbors
+
+- Add `neighbors.SWINN` to power-up approximate nearest neighbor search. SWINN uses graphs to speed up nearest neighbor search in large sliding windows of data.
+
```

Comment:
@smastelini you could also explain that KNNRegressor and KNNClassifier can be passed different NN backends

Reply:
Good catch! I will do that :D

Reply:
Done!